packages/@uppy/companion-client/src/RequestClient.js

```diff
@@ -58,7 +67,7 @@ export default class RequestClient {
   }
 
   headers () {
-    const userHeaders = this.opts.companionHeaders || {}
+    const userHeaders = this.#companionHeaders || {}
```

Comment:
We probably don't need the variable, we can pass the value directly to the spread operator.
```suggestion
```